seqr/views/utils/anvil_metadata_utils.py

```diff
@@ -492,6 +492,7 @@ def _get_genetic_findings_rows(rows: list[dict], individual: Individual, partici
                 }[zygosity],
                 'allele_balance_or_heteroplasmy_percentage': heteroplasmy,
                 'variant_inheritance': _get_variant_inheritance(individual, genotypes),
+                'ClinGen_allele_ID': row.pop('CAID', None),
```

Comment:
This line means that ClinGen_allele_ID is in all genetic findings reports, whether or not we asked for CAID in the initial query. Is this the wrong approach @hanars ?

Reply:
we should have this column in all the genetics findings reports, there should not be any conditional logic around it

Reply:
The column `ClinGen_allele_ID` is in all genetics findings reports. This line stays the same, the `None` default is there in case `CAID` doesn't exist in the saved variant json.